temporal-sdk/src/main/java/io/temporal/internal/sync/WorkflowThreadContext.java

```diff
@@ -216,11 +215,14 @@ public String getYieldReason() {
   }
 
   /**
+   * @param deadlockDetectionTimeoutMs max duration is ms a workflow code can run without yielding
+   *                                   before it's considered deadlocked.
+   *                                   Discarded if Temporal is configured in debug mode.
```

Comment:
I would state the environment variable name in this comment instead of "in debug mode".